packages/flutter/test/material/date_picker_test.dart

```diff
@@ -774,3 +777,70 @@ void _tests() {
     expect(tester.getBottomLeft(find.text('OK')).dx, 800 - ltrOkRight);
   });
 }
+
+void _screenConfigTests() {
+
+  const Size kPixel1Portrait = Size(1070, 1770);
+  const Size kPixel1Landscape = Size(1770, 1070);
+  const Size kSmallScreenPortrait = Size(320, 521);
+  const Size kSmallScreenLandscape = Size(521, 320);
+
+  Future<void> _showPicker(WidgetTester tester, Size size, [double textScaleFactor = 1.0]) async {
+    tester.binding.window.physicalSizeTestValue = size;
+    tester.binding.window.devicePixelRatioTestValue = 1.0;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Builder(
+          builder: (BuildContext context) {
+            return RaisedButton(
+              child: const Text('X'),
+              onPressed: () {
+                showDatePicker(
+                  context: context,
+                  initialDate: DateTime(2018, DateTime.december, 30),
+                  firstDate: DateTime(2018),
+                  lastDate: DateTime(2030),
+                );
+              },
+            );
+          },
+        ),
+      ),
+    );
+    await tester.tap(find.text('X'));
+    await tester.pumpAndSettle();
+  }
+
+  testWidgets('display on Pixel1 - portrait', (WidgetTester tester) async {
+    await _showPicker(tester, kPixel1Portrait);
```

Comment:
These tests are verifying that we don't assert. Just to make that clear it would be good to append:
```
 expect(tester.takeException(), isNull);
```

Reply:
Nice suggestion. I have added this to all the tests.